docs/data-tunnel/route-types/02-ibc.md

```diff
@@ -0,0 +1,100 @@
+# IBC Route
+
+## Overview
+
+The **IBC Route** is one of the tunnel routes used to transmit tunnel packet data from **BandChain** to other **Cosmos SDK-based blockchains**. To utilize this route, consumers have two options:
+
+1. **Core Application Implementation**: Implement the `OnRecvPacket` function within the core application. This method requires Cosmos applications to update their chain to support the IBC tunnel.
+2. **WASM Contract Implementation**: Implement a WASM contract to support the IBC tunnel. This method requires Cosmos applications to support WASM contracts.
+
+## Creating an IBC Tunnel
+
+To create an IBC tunnel using the command-line interface (CLI), execute the following command:
+
+```bash
+bandd tx tunnel create-tunnel ibc [initial-deposit] [interval] [signalInfos-json-file]
+```
+
+### Parameters:
+
+- **initial-deposit**: The initial deposit amount required to create the tunnel.
+- **interval**: The time interval at which data updates are sent.
+- **signalInfos-json-file**: A JSON file containing signal information for the tunnel.
+
+This command initializes an IBC tunnel that facilitates secure and efficient transmission of tunnel packet data from BandChain to other Cosmos SDK-based blockchains.
+
+## Updating Route Information
+
+BandChain allows tunnel creators to edit some route details. In this case, the tunnel creator can update the IBC channel ID by executing the following command:
+
+```bash
+bandd tx tunnel update-route ibc [tunnel-id] [channel-id]
+```
```

Comment:
For IBC, the user is required to create an IBC channel and update the route information. it's not optional. we may need to update that info and how to create channel a bit.